jest.a11y.config.js

```diff
@@ -1,14 +1,12 @@
 module.exports = {
   roots: ['<rootDir>/src/test/a11y'],
-  "testRegex": "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
-   "moduleFileExtensions": [
-    "ts",
-    "js"
-  ],
-  "testEnvironment": "node",
+  testRegex: "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
+  testEnvironment: "node",
   transform: {
     '^.+\\.ts?$': 'ts-jest',
   },
-  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
-  setupFiles: ["<rootDir>/jestEnvVars.js"]
+  moduleFileExtensions: ['ts', 'js'],
+  setupFiles: ["<rootDir>/jestEnvVars.js"],
+  // pa11y default
```

Comment:
```suggestion
  // pa11y recommended default
```
if it was the pa11y default we wouldnt have to set it ;)